packages/browser/jest.config.js

```diff
@@ -12,4 +12,7 @@ module.exports = createJestTSConfig({
       statements: 87.25,
     },
   },
+  moduleNameMapper: {
+    'customerio-gist-web': '<rootDir>/../browser/src/plugins/in-app-plugin/__tests__/fake-gist.ts',
```

Comment:
There's probably a better way to do this, but I couldn't get Gist under test via jest without a full mock like this. Always encountered this error:

```
  SyntaxError: Cannot use import statement outside a module
 
        4 |
        5 | import { InAppEvents, semanticEvents, newEvent, allEvents, gistToCIO } from './events'
     > 6 | import Gist from 'customerio-gist-web'
```

Definitely open to suggestions. I did try changing https://github.com/customerio/gist-web/ from default to named exports locally, but no luck.